src/plugins/discover/public/application/components/sidebar/discover_field_bucket.tsx

```diff
@@ -45,13 +53,29 @@ export function DiscoverFieldBucket({ field, bucket, onAddFilter }: Props) {
     }
   );
 
+  const tooltipContent = `${bucket.value}: ${bucket.count} (${bucket.percent}%)`;
+
   return (
     <>
-      <EuiFlexGroup gutterSize="xs" responsive={false}>
-        <EuiFlexItem className="eui-textTruncate">
-          <EuiText size="xs" className="eui-textTruncate">
-            {bucket.display === '' ? emptyTxt : bucket.display}
-          </EuiText>
+      <EuiFlexGroup justifyContent="spaceBetween" responsive={false} gutterSize="s">
+        <EuiFlexItem className="dscFieldDetails__barContainer" grow={1}>
+          <EuiToolTip content={tooltipContent} delay="regular" position="right">
```

Comment:
Setting the `delay' to `long` made the functional test pass locally, there was some overlapping because of the tooltip. also I thing a bit more delay here is better.

Reply:
Well, seems this doesn't work at Jenkins and furthermore be potential source for flaky tests, will think about an alternative later on